js/src/huobi.js

```diff
@@ -2945,7 +2945,7 @@ export default class huobi extends Exchange {
         }
```

Comment:
@ttodua can you check this PR and validate the if these changes are correct?

Reply:
@carlosmiei i dont think it's easy to validate that. in previous version, network codes were not fully unified, and my pending networks-pr does all-around checks to completely address all network related things. the proposed change by OP might fix a minor case (might be patch for something) but might break something other in related exchanges.
that's why I think it's better to wait a few more days until someone from @ lead would find a time to review the PR that I know fixes and handles all exchanges in a way, where none of them are broken. I would avoid touching this minor modification and suggest user to make an override of that method in his userland code, until we have a complete solution.

Reply:
but if you think we should address this atm, then I'll take a look into this previous implementation.

Reply:
@ttodua Fix works great for me, but I didn't really expect this to be merged for reasons you mentioned. Just giving a heads up for someone who knows codebase in and out.

Reply:
I'm about to submit similar fix for okx which is also broken

Reply:
dmatora, thanks for contribution!
I hope in the upcoming week we will get through the complete networks unification PR which has been pending for a long already.